cart/src/main/java/com/yas/cart/model/Cart.java

```diff
@@ -20,6 +20,8 @@ public class Cart extends AbstractAuditEntity {
 
     private String customerId;
 
+    private Long orderId = null;
```

Comment:
We don't need to set null to `orderId`, If don't set value var has a Long type always is null

Reply:
i'll update